storage/innobase/include/btr0scrub.h

```diff
@@ -0,0 +1,14 @@
+// Copyright 2014 Google
+
+#ifndef btr0scrub_h
+#define btr0scrub_h
+
+#include "univ.i"
+
+#include "btr0types.h"
+#include "data0data.h"
+#include "dict0dict.h"
+#include "mtr0mtr.h"
+#include "page0cur.h"
+
+#endif
```

Comment:
Same reason as for the source file related to it: it will contain more features with the other scrubbing parts, I just split it up into smaller commits.